line-sdk/src/main/java/com/linecorp/linesdk/internal/nwclient/core/ChannelServiceHttpClient.java

```diff
@@ -332,6 +357,11 @@ private static void setRequestHeaders(
                                 httpResponseCode,
                                 errorResponseParser.getResponseData(inputStream)));
             }
+
+            if (responseDataParser == null) {
+                return LineApiResponse.createAsSuccess(null);
+            }
```

Comment:
`errorResponseParser` is declared as `@NonNull`, we don't need to check `null`  here. If it could be `null`, we need to remove `@NonNull` annotation.

Reply:
In one case of Openchat implementation, the response is empty so no need to put responseDataParser as argument. I will make the annotation from `@NonNull` to `@Nullable`